docs/cita/sys-contract-interface/interface.md

```diff
@@ -1166,6 +1166,7 @@ $ cita-cli scm QuotaManager setDefaultAQL \
 * 参数
 
     `uint` - The value to be setted
+    `address` - The address to be setted
```

Comment:
2 problem:

1. `setted` is a typo, should use `set`.
2. `The address to be setted` means nothing.  Should explain what kind of address can be set, an account address or a contract address?

Reply:
@rainchen fixed